src/components/pages/ImageClassification/useSelectNewAttribute.js

```diff
@@ -1,32 +1,11 @@
-import React, { useState } from 'react'
-import PropTypes from 'prop-types'
+import { useState } from 'react'
```

Comment:
This is a weird Diff. Git sees it as a file rename. I created a new hook and moved the non-JSX part into it and deleted the old (`NewRow`) component.